File: app/src/main/java/com/vpulse/ftpnext/commons/interfaces/OnEndAnimation.java
package com.vpulse.ftpnext.commons.interfaces;

import android.view.animation.Animation;

public abstract class OnEndAnimation implements Animation.AnimationListener {

    @Override
    public void onAnimationStart(Animation animation) {

    }

    @Override
    public abstract void onAnimationEnd(Animation animation);

    @Override
    public void onAnimationRepeat(Animation animation) {

    }
}

File: app/src/main/java/com/vpulse/ftpnext/commons/interfaces/OnEndAnimation.kt
package com.vpulse.ftpnext.commons.interfaces

import android.view.animation.Animation

abstract class OnEndAnimation : Animation.AnimationListener {
    override fun onAnimationStart(animation: Animation) {}
    abstract override fun onAnimationEnd(animation: Animation)
    override fun onAnimationRepeat(animation: Animation) {}
}